### PR TITLE
#184752229  finish the setup of unit tests and a simple addition test

### PR DESCRIPTION
**Pull request by Rod-code**

### CODE REVIEW

File: tests/addition/addition.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+function add(a, b) {
+    return a + b;
```
Comment (nshutijonathan) [inline tests/addition/addition.js:2]: is this necessary?

File: tests/addition/addition..test.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+const add = require('./addition');
+
```
[the remark this answers is not in the document]
Comment (Rod-code) [reply tests/addition/addition..test.js:2]: i just wanted to add a simple test to check if tests are working successfully

File: tests/addition/addition.js (hunk 1 above)
Comment (nshutijonathan) [reply tests/addition/addition.js:2]: remain with only one

**Review by nshutijonathan — APPROVED** [review]

**Review by EdwinBayingana — APPROVED** [review]
Looks good now.

**Review by David-Neza — APPROVED** [review]
Looks good

**Review by madebypatrick — APPROVED** [review]

**Pull request merged**